### ISSUE TITLE
MB-Melgan loss increasing at 200k steps

### ISSUE THREAD

**Issue by wsstriving** [kan-bayashi/ParallelWaveGAN#226]
Hi,
      When I ran the recipe for VCTK using MB-Melgan, I found that the losses increase at the 200ksteps. I checked the pretrained model you provided, the curve is similar. I would like to ask, is it normal and how much impact will be caused on the wave generation?
     Thanks a lot!
![image](https://user-images.githubusercontent.com/9416457/97826052-83e31b00-1cfb-11eb-9d30-0e999e861323.png)

**Comment by kan-bayashi** [comment]
I also confirmed the similar training curve.
Maybe there is a room for the improvement by changing the LR scheduler decay timing.
But I've not yet investigated. If you share your experiments, that is great :)

**Comment by LLianJJun** [comment]
I think it is because discriminator training started after 200k.

**Comment by kan-bayashi** [comment]
Oh, I misunderstood your post.
Jumping at 200k is caused by the introduction of the discriminator.
Jumping at 400k or 600k is caused by the lr scheduling.
Consequently, the jumping at 200k is no problem, that is normal behavior.
